### PR TITLE
minor support for generic-pool logging

### PR DESCRIPTION
**Pull request by ghost**
Logging is hard.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "node-thrift-pool",
-  "version": "1.0.4",
+  "name": "gaiden-thrift-pool",
```
Comment (azylman) [inline package.json]: Can you revert the changes to this file?

**Comment by azylman** [comment]
@turristan If you revert the changes to the package.json file, I'll merge + publish

**Comment by ghost** [comment]
Reverted.
Oh, well, I used to thought pr commits are fixed.

**Comment by azylman** [comment]
@turristan published as 1.1.0
